Berksfile

```diff
@@ -3,21 +3,21 @@ source 'https://supermarket.chef.io'
 
 metadata
 
-cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
-cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
+cookbook 'conda', github: "robzor92/conda-chef", branch: "HOPSWORKS-1809"
```

Comment:
Make sure you revert these changes before merging.

Reply:
Sure